cfg/aws/credentials.go

```diff
@@ -60,9 +60,8 @@ var credentialsChain = make([]RootCredentialsProvider, 0)
 
 func getRootCredentialsFromChain(c *CredentialConfig) *credentials.Credentials {
 	for _, provider := range credentialsChain {
-		credentials := provider.Credentials(c)
```

Comment:
I prefer the formatting of the first but both are okay.

Reply:
I'm mostly indifferent. I don't think it is really "cleaner" one way or the other